sympy/parsing/c/c_parser.py

```diff
@@ -365,16 +391,46 @@ def transform_function_decl(self, node):
 
 
             """
-            token = node.get_tokens()
-            c_ret_type = next(token).spelling
-            if (c_ret_type == 'void'):
-                ret_type = none
-            elif(c_ret_type == 'int'):
-                ret_type = IntBaseType(String('integer'))
-            elif (c_ret_type == 'float'):
-                ret_type = FloatBaseType(String('real'))
+            tokens_spelling = [
+            token.spelling for token in node.get_tokens()]
```

Comment:
Also, do these definitions really need to be in two lines? this is just increasing the size of the file.

Reply:
I thought to follow PEP8 strictly, that line exceeded 72 characters :sweat_smile: